frontend/packages/console-dynamic-plugin-sdk/src/shared-modules.ts

```diff
@@ -11,6 +11,11 @@ export const sharedVendorModules = [
   'react-i18next',
   'react-router',
   'react-router-dom',
+  '@patternfly/patternfly',
+  '@patternfly/react-core',
+  '@patternfly/react-table',
+  '@patternfly/react-icons',
+  '@patternfly/react-styles',
```

Comment:
@cyril-ui-developer Let's try with this smaller set and see if everything works.

```suggestion
  '@patternfly/react-core',
  '@patternfly/react-table',
```